apps/ledger-live-mobile/src/components/RootNavigator/DiscoverNavigator.tsx

```diff
@@ -20,21 +20,32 @@ export default function DiscoverNavigator() {
   return (
     <Stack.Navigator screenOptions={stackNavigationConfig}>
       {config?.enabled && config?.params.version === "2" ? (
-        <Stack.Screen
-          name={ScreenName.PlatformCatalog}
-          component={Catalog}
-          options={{
-            headerShown: false,
-          }}
-        />
+        <>
+          <Stack.Screen
+            name={ScreenName.PlatformCatalog}
+            component={Catalog}
+            options={{
+              headerShown: false,
+            }}
+          />
+        </>
       ) : (
-        <Stack.Screen
-          name={ScreenName.DiscoverScreen}
-          component={Discover}
-          options={{
-            headerShown: false,
-          }}
-        />
+        <>
+          <Stack.Screen
+            name={ScreenName.DiscoverScreen}
+            component={Discover}
+            options={{
+              headerShown: false,
+            }}
+          />
+          <Stack.Screen
+            name={ScreenName.PlatformCatalog}
+            component={Catalog}
+            options={{
+              headerShown: false,
+            }}
+          />
+        </>
       )}
```

Comment:
Simply as this?
```javascript
<Stack.Navigator screenOptions={stackNavigationConfig}>
  {config?.enabled && config?.params.version === "2" && (
    <Stack.Screen
      name={ScreenName.DiscoverScreen}
      ..
    />
  )}
  <Stack.Screen
    name={ScreenName.PlatformCatalog}
    ..
  />
</Stack.Navigator>
```

Reply:
This is what was originally coded, but ios or android have different behaviours and don't work as expected
With this ios still open the Menu